src/Mert/TicketBundle/Form/TicketType.php

```diff
@@ -5,21 +5,26 @@
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Bridge\Doctrine\Form\Type\EntityType;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
 
 class TicketType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
 
-        $builder->add('category', 'entity', array('class' => 'Mert\TicketBundle\Entity\Category'));
-        $builder->add('title', 'text', array('label' => false));
-        $builder->add('content', 'textarea', array('label' => false));
-        $builder->add('priority', 'choice',
+        $builder->add('category', EntityType::class, array('class' => 'Mert\TicketBundle\Entity\Category'));
```

Comment:
Thanks for the support, only i see, you are removing this entity but still using in here.

Reply:
User Entity is deleted a double file which has the extension "php ~" the original file


```
$builder->add('category', EntityType::class, array('class' => 'Mert\TicketBundle\Entity\Category'));
```
This is read Entity Category

Reply:
Ah, i see now, thank you